### PR TITLE
Authentication created and needs to be checked

### PR DESCRIPTION
**Pull request by sifulsidd**

### CODE REVIEW

**Review by Ericesposito — APPROVED** [review]

**Review by paglenn — CHANGES_REQUESTED** [review]
can you remove the .env?

**Review by brok3turtl3 — APPROVED** [review]
Nice work on Auth!

File: server/routes/userRoutes.js [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
 
-router.post("/", registerUser, (req, res) => {
-  res.status(200).send("Success!!!");
-});
+const router = express.Router();
 
-router.post("/login", authUser, () => {});
+import {userLogin, createUser} from '../controllers/userController';
 
-export default router;
```
Comment (joie-zhang) [inline server/routes/userRoutes.js:13]: It looks like the router is no longer being exported!

File: server/controllers/userController.js [hunk 2]
```diff
@@ -1,13 +1,93 @@
-//ENDPOINT  POST api/users
-//PURPOSE   Register a new user
-//ACCESS    Public
```
Comment (joie-zhang) [inline server/controllers/userController.js:3]: could we get that pseudocode copied over by any chance?

**Review by paglenn — APPROVED** [review]

File: server/controllers/userController.js [hunk 3]
```diff
@@ -1,13 +1,93 @@
...
+            status: 500,
+            message: { err: 'An error occurred during verifying token' },
+          });
+    }
+  }
 };
 
-//ENDPOINT  POST api/users/login
-//PURPOSE   Authenticate User and get token
-//ACCESS    Public
```
Comment (joie-zhang) [inline server/controllers/userController.js:10]: and this pseudocode as well?

**Review by joie-zhang — APPROVED** [review]
Cool!

**Pull request merged**